### PR TITLE
Add blog post for Cake 2.1.0

### PR DESCRIPTION
**Pull request by augustoproiete**
- v2.1.0 Blog Post

### CODE REVIEW

File: input/blog/2022-02-19-cake-v2.1.0-released.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+releaseName: 2.1.0
+---
+
+Version **2.1.0** of Cake has been released. Take it for a spin and give us feedback on our [discussion board](https://github.com/cake-build/cake/discussions/3818).
+
+This release includes new features, improvements and bug fixes to both [Cake Scripting](/docs/running-builds/runners/dotnet-tool) and [Cake Frosting](/docs/running-builds/runners/cake-frosting) since the [Cake v2.0.0 release](/blog/2021/11/cake-v2.0.0-released)! 🚀 🍰
+
+### Highlights of this release
+
+- The new aliases for using `dotnet format` and `dotnet sdk` in Cake builds
```
Comment (pascalberger) [inline input/blog/2022-02-19-cake-v2.1.0-released.md:14]:
```suggestion
- New aliases for using `dotnet format` and `dotnet sdk` in Cake builds
```

File: input/blog/2022-02-19-cake-v2.1.0-released.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- New alias for the Chocolatey Export command
+- `XmlTransform` now supports XSL arguments
+- Several improvements to GitHub Actions, including support for the the `DownloadArtifact` command
```
Comment (pascalberger) [inline input/blog/2022-02-19-cake-v2.1.0-released.md:17]:
```suggestion
- Several improvements to GitHub Actions, including support for the `DownloadArtifact` command
```

**Review by pascalberger — APPROVED** [review]
Some small typos. Otherwise LGTM

**Comment by augustoproiete** [comment]
> Some small typos. Otherwise LGTM

Thanks. Fixed typos

**Review by gep13 — APPROVED** [review]
LGTM!

**Review by devlead — APPROVED** [review]
LGTM 👍

**Pull request merged**